### PR TITLE
Allow original email's Subject to be included in bounce message

### PR DESCRIPTION
**Pull request by davebeyer**
Add the original email's Subject to the set of values that can be used in the config/outbound.bounce_message template, for more informative bounce messages.

### CODE REVIEW

File: outbound.js [hunk 1]
```diff
@@ -1709,6 +1709,7 @@ HMailItem.prototype.populate_bounce_message_with_headers = function(from, to, re
         me:   config.get('me'),
         from: from,
         to:   to,
+        subject: header.get('Subject').trim(),
```
Comment (baudehlo) [inline outbound.js]: This should use `.get_decoded()` otherwise it's in the original format.

**Comment by davebeyer** [comment]
@baudehlo, thanks for the tip.  I made the change but unfortunately it now seems to be failing one of the integration checks.  (Is there a way for me to see what check is failing?)    After this is resolved, if the change seems reasonable to you, I'm happy to rebase to clean it up.

**Comment by baudehlo** [comment]
Seems to be a simple deprecation problem in our eslint settings. @msimerson can you figure out what the issue is?

**Comment by msimerson** [comment]
Okay, HEAD is updated for compatibility with eslint 2. If you rebase against haraka/master, the integration tests should now pass.

**Comment by davebeyer** [comment]
Thanks @msimerson - I've rebased and integration tests now pass.

**Comment by msimerson** [comment]
+1

**Comment by Dexus** [comment]
:+1:  thanks

**Pull request merged**